shlr/tree_sitter_r2cmd.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```

Comment:
imho we should have a generic script to fetch external repos, rather than copying the same script with almost the same code. we can maybe reuse it for other things.. but all that stuff was done in makefiles for capstone, no need for shellscripts maybe

Reply:
capstone does use a script AFAIK (capstone.sh).

I can use just one script for tree-sitter and tree-sitter-r2cmd. Capstone is a bit different because it requires patching and other complicated stuff :/